src/appdev/ChronicleSession.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import { config, S3 } from 'aws-sdk';
```

Comment:
wait a second, why'd you copy & paste this in? i thought we were publishing `appdev` to `npm`?

Reply:
More cryptic errors. I believe the issue is related to how Webpack is creating the bundle or how Jest runs tests on the bundle when used in conjunction with certain dependencies/libraries, same as the Axios bug we had (technically still have since we just replaced Axios altogether) that would crash the app because of an undefined variable error. I even wrote a custom branch of a library that `appdev` uses, added it, and wrote `appdev` tests to make sure it worked, but still no luck getting the `appdev` library to work with Transit. Then I more or less copy-pasted the files as you can see and it worked, so unfortunately I think it needs to stay like this unless someone is capable of fixing or even identifying the underlying issue that's causing certain dependencies to not work properly.

Reply:
To be clear the `appdev` project works fine it's just not working with Transit specifically.